tests/KdybyTests/Console/Parameters.phpt

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * Test: Kdyby\Console\ContainerHelper.
+ *
+ * @testCase KdybyTests\Console\ParametersTest
+ * @author Martin Procházka <juniwalk@outlook.cz>
+ * @package Kdyby\Console
+ */
+
+namespace KdybyTests\Console;
+
+use Kdyby;
+use Nette;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Tester\ApplicationTester;
+use Tester;
+use Tester\Assert;
+use Tracy\Debugger;
+
+
+
+require_once __DIR__ . '/../bootstrap.php';
+
+/**
+ * @author Martin Procházka <juniwalk@outlook.cz>
+ */
+class ParametersTest extends \Tester\TestCase
+{
+
+	private function prepareConfigurator()
+	{
+		$config = new Nette\Configurator();
+		$config->setTempDirectory(TEMP_DIR);
+		$config->addParameters(array('container' => array('class' => 'SystemContainer_' . Nette\Utils\Strings::random())));
+		Kdyby\Console\DI\ConsoleExtension::register($config);
+		Kdyby\Events\DI\EventsExtension::register($config);
+		$config->addConfig(__DIR__ . '/config/parameters.neon', $config::NONE);
+
+		return $config;
+	}
+
+
+
+	public function testParametersCommand()
+	{
+		Debugger::$logDirectory = TEMP_DIR . '/log';
+		Tester\Helpers::purge(Debugger::$logDirectory);
+
+		/** @var Nette\DI\Container $container */
+		$container = $this->prepareConfigurator()->createContainer();
+
+		/** @var Kdyby\Console\Application $app */
+		$app = $container->getByType('Kdyby\Console\Application');
```

Comment:
You probably don't even have to mock the container, just use `new Nette\DI\Container()` and pass some arguments and the do asserts on the helper